packages/core-sdk/tests/exchangePolicy/exchangePolicyRules.production.json

```diff
@@ -0,0 +1,107 @@
+{
+  "yupSchema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/person.schema.json",
+    "title": "Person",
+    "description": "A person",
```

Comment:
I guess these values are not used? otherwise change them I guess

Reply:
done